website/docs/registries/overview.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+Scarb is not only a Cairo build toolchain but also a package manager.
+It uses registries to store and manage packages.
+While Scarb supports alternative ways to host packages, registries make it easy to publish, discover, and integrate packages into your projects.
+
+## Official registry
+
+Currently, the default way to host packages is via the official [scarbs.xyz](https://scarbs.xyz) registry.
+Please note that the official registry is still in development.
+You can already use it to discover and [add](#adding-dependencies) packages to your projects.
+[Publishing](#packaging-and-publishing) packages is currently limited, but if there are any other packages you would like to be available - please reach out to us on [Telegram](https://t.me/scarbs_xyz) or [Discord](https://discord.gg/7YXj4Z2).
```

Comment:
This should be stated as an info "bubble" above so it's read first, and can be comfortably removed later

Reply:
Can you please clarify what exactly do you mean by "above"?

Would seem strange to introduce one missing feature prior to introducing the registry itself.

Reply:
By above i mean at the top of this doc, under the title

It's a limitation that's really important to understand prior to reading any content about the registry. It tells about the stage that it's at.

It's the most important information here for the user, far more important than telling him it's in development, or that they can explore packages.

Reply:
> By above i mean at the top of this doc, under the title

While this is not my PR originally, it is my understanding that the idea of these docs was to describe registries as a concept, not exclusively the official registry, which is why (probably) the section is titled *registries* and not *scarbs.xyz*, so I don't think I can really include it at the top of the doc, as `Overview` isn't specifically about the official registry.

> It's the most important information here for the user, far more important than telling him it's in development, or that they can explore packages.

While we can consider adding it as a "bubble" under `Official registry`, I believe the average user is more interested in downloading packages rather than sharing them, and in knowing what they *can do*, not what they can't. 

Imo anyone interesting in publishing their package will probably be willing one extra sentence.